functions/functionsv2/helloauditlog/hello_auditlog.go

```diff
@@ -0,0 +1,52 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START functions_log_cloudevent]
+
+// Package helloworld provides a set of Cloud Functions samples.
+package helloworld
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"github.com/cloudevents/sdk-go/v2/event"
+	auditevents "github.com/googleapis/google-cloudevents-go/cloud/audit/v1"
+)
+
+// HelloAuditLog receives a Cloud Audit Log event, and logs a few fields.
```

Comment:
suggestion: process -- when tracking against a canonical sample it can help to link the canonical in the PR description. https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/2991f86146742d00725c4dd73bf3719bd25e809b/functions/v2/index.js#L65

Reply:
Done. ive added the link to the PR description. Thanks for providing the link!